ui/client/types/process.ts

```diff
@@ -4,6 +4,7 @@ import {ValidationResult} from "./validation"
 
 export type Process = {
   id: string,
+  newId?: string,
```

Comment:
do we have to put it here? currently this type reflects the data that come from the backend...